### DISCUSSION TITLE
Looks strange if you have a list of private repos and nothing is showing to people who can't see them

### DISCUSSION THREAD

**Discussion by lindsayesterman** [community/community#12323] [Other Feature Feedback, Questions, & Ideas]
Maybe a sentence that says "the repos in this list are private" so it doesn't look like an empty list.

**Comment by austenstone** [comment]
I think the group should only appear if it has `> 0` repos that the viewer can see.
If the list is full of all private repositories then hide it from the viewer unless the viewer has access to those private repositories.

I don't want to even share the title of these private work related lists with the public.

**Comment by Rock1a** [comment]
Thanks 👍
